dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -153,21 +157,50 @@ def dataset_compatible_tensorflow():
   class _DALIDatasetV2(dataset_ops.DatasetSource):
     def __init__(
       self,
-      pipeline = '',
+      pipeline,
+      output_dtypes,
+      output_shapes = None,
+      *,
```

Comment:
wondering, what is this `*`?

Reply:
After this all arguments are strictly keyword arguments and can't be supplied as positional arguments.